cms/templates/cms/program_page.html

```diff
@@ -4,7 +4,7 @@
 {% load wagtailcore_tags %}
 {% load wagtailimages_tags %}
 
-{% block title %}{{ page.title }} MicroMasters{% endblock %}
+{% block title %}{{ page.title }}{% endblock %}
```

Comment:
This is also OK as is. It's useful to append 'MicroMasters' to the program name.